enterprise/app/settings/github_link.tsx

```diff
@@ -217,18 +243,38 @@ export default class GitHubLink extends React.Component<Props, State> {
                 <div>
                   <div className="github-app-installations">
                     {this.state.installationsResponse?.installations.map(
-                      (installation) => (
+                      (installation, idx) => (
                         console.log(installation),
                         (
                           <div className="github-app-installation">
-                            <div className="installation-owner">
-                              <TextLink href={`https://github.com/${installation.owner}`}>
-                                {installation.owner}
-                              </TextLink>
+                            <div className="installation-header">
+                              <div className="installation-owner">
+                                <TextLink href={`https://github.com/${installation.owner}`}>
+                                  {installation.owner}
+                                </TextLink>
+                              </div>
+                              <OutlinedButton
+                                className="destructive"
+                                onClick={() => this.showUnlinkDialog(installation)}>
+                                Unlink
+                              </OutlinedButton>
+                            </div>
+                            <div className="installation-settings">
+                              <div className="input-row">
+                                <label className="input-label">
+                                  <Checkbox
+                                    checked={installation.reportCommitStatusesForCiBuilds}
+                                    onChange={(e) =>
+                                      this.updateInstallationSettings(installation, idx, e.target.checked)
+                                    }
+                                  />
+                                  <span>Report commit statuses to GitHub</span>
+                                </label>
+                              </div>
+                              <div className="input-description">
+                                Applies to Workflows and builds tagged with Role=CI
```

Comment:
nit: capitalize `ROLE` to match how it's set in build metadata; also maybe consider linking to the relevant docs on build metadata like `<TextLink href="https://www.buildbuddy.io/docs/guide-metadata#role">ROLE=CI</TextLink>`, since I think most people wouldn't know what this ROLE=CI is referring to when they're visiting this page and likely just getting started.

```suggestion
                                Applies to BuildBuddy Workflows and builds tagged with <TextLink href="https://www.buildbuddy.io/docs/guide-metadata#role">ROLE=CI</TextLink>
```